docset/windows/storage/get-physicaldisk.md

```diff
@@ -193,14 +193,16 @@ Accept wildcard characters: False
 ```
 
 ### -HealthStatus
-Specifies the health status of physical disks to get.
+Specifies the **Health Status** of physical disks to get.
 The acceptable values for this parameter are:
 
 - Healthy 
 - Unhealthy 
 - Unknown 
 - Warning
 
+Refers to the SMART counters of all Hard drives has, and represents a wait to get some failure prediction so you can generate de actions to change or repair the disk.
```

Comment:
Suggestion:
**Refers to the SMART counters present in all hard drives. This information can be used to take action to, for example, replace or repair a disk.**

Reply:
Alternate suggestion:
**Refers to the SMART counters of all hard drives, generating a failure prediction that can help you take action to change or repair the disk.**